Makefile

```diff
@@ -2,35 +2,33 @@ default_target: local
 
 COMMIT_HASH := $(shell git log -1 --pretty=format:"%h"|tail -1)
 VERSION = 0.12.0
+IMAGE_OWNER ?= blakeblackshear
```

Comment:
Maybe instead of `IMAGE_OWNER` this should be `IMAGE_REPO` with the default set to `ghcr.io/blakeblackshear/frigate`

Reply:
Done.